src/main/java/org/cbioportal/proxy/ProxyController.java

```diff
@@ -226,54 +213,30 @@ private <T> ResponseEntity<T> exchangeData(String body, URI uri, HttpMethod meth
         return restTemplate.exchange(uri, method, new HttpEntity<>(body, httpHeaders), responseType);
     }
 
+    private boolean isKnownHost(String pathInfo) {
+        String[] knownHosts = {
+            this.genomeNexusUrl,
+            this.genomeNexusGrch38Url,
+            "http://mutationaligner.org/api",
```

Comment:
do we still use mutationaligner?

Reply:
Looks like it, just to check for existence of this URL:

![image](https://github.com/cBioPortal/cbioportal/assets/1334004/7db30213-a2d2-43e2-9aa0-4792bbdaad0c)

Reply:
We decided to always show the link. See cBioPortal/cbioportal-frontend#4942